wls-exporter-core/src/main/java/com/oracle/wls/exporter/UrlBuilder.java

```diff
@@ -20,26 +22,48 @@
  */
 public class UrlBuilder {
 
-  private static final Set<Integer> successes = new ConcurrentSkipListSet<>();
+  private static class WebHost implements Comparable<WebHost> {
+    private final String hostName;
+    private final int port;
+
+    public WebHost(String hostName, int port) {
```

Comment:
would it be better to name this inner class such as WebHostPort?

Reply:
I did some research into this. Apparently, to the extent that a standard exists, the name/port combination is called the "host" and the name alone is supposed to be called, "host name." So I renamed the variables appropriately. 

https://stackoverflow.com/questions/13673389/difference-between-host-and-hostname-in-window/13673475